dbConnection/mongo.js

```diff
@@ -0,0 +1,14 @@
+const mongoose = require('mongoose');
+ // Set a Mongoose Promise Library
+mongoose.Promise = global.Promise;
+// const dbpassword = process.env.DB_PASSWORD; <- for mlab
+// const dbuser = process.env.DB_USER; <- for mlab
+const dbURI = process.env.MONGODB_URI || 'mongodb://localhost:27017/outbooked-demo';
+mongoose.connect(dbURI, { useNewUrlParser: true });
+mongoose.connection.once('open', () => {
+  console.log('Mongo Connected.')
+});
+mongoose.connection.on('error', console.error.bind(console, 'MongoDB connection Error:'));
+mongoose.set('useCreateIndex', true);
+mongoose.set('useFindAndModify', false); // corrects the annoying DeprecationWarning
+// mongoose.set('debug', true);
```

Comment:
I appreciate that you did the Mongo connection, but considering that we were asked to do a MySQL connection, and you did a MySQL connection in this same branch, I don't see a need for having the Mongo connection here.

Reply:
I'm using mongoDB for testing purposes initially then switch over to the SQL side once we know what to look for in the redirect and what we need to store sense we are more comfortable with mongo...